docs/Administrator Guide/SSL.md

```diff
@@ -60,6 +60,11 @@ the same resource, because either would be insecure.  Instead, any such "mixed
 mode" connections will be rejected by the TLS-using side, sacrificing
 availability to maintain security.
 
+One point to note is that the TLS certificate verification will fail if the
```

Comment:
Suggest that you do away with the "One point to note is that" part of this sentence.

Reply:
Ack. Will trim the sentence accordingly.